build/yamls/antrea-eks.yml

```diff
@@ -541,6 +556,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: spec.nodeName
+        - name: API_PORT
+          value: "10350"
```

Comment:
ditto

Reply:
Relied in `pkg/antctl/raw/bundle/command.go` thread.